main.py

```diff
@@ -73,7 +73,32 @@
 
     await bot.process_commands(message)
 
+#start CuckTim Section
+#replaces "Cuck" with Rotating Light Emoji, meant to piss off Tim
+@bot.event
+async def on_message(message):
+    if not message.author.bot and "cuck" in message.content.lower():
+        CuckString = message.content.replace('cuck', ':rotating_light:')
+        newString = re.compile("cuck", re.IGNORECASE)
+        await bot.send_message(message.channel, newString.sub('cuck', cuckString))
+    elif (str(ctx.message.author) == "RacistJudicata#8536"):
+        await bot.say("Stop it, Tim")
+    else:
+         await bot.process_commands(message)
+
+#end CuckTim Section
 
+#start /r/replacement
```

Comment:
Let's remove this code for now and focus on the 'cuck' thing above this for now. Once we get through that, we can come back around on this.